extensions/colorslurp/src/utilities.ts

```diff
@@ -0,0 +1,24 @@
+import { getApplications, showToast, Toast, open, closeMainWindow } from "@raycast/api";
+
+async function isColorSlurpInstalled() {
+  const applications = await getApplications();
+
+  return applications.some(({ bundleId }) => {
+    return bundleId === "com.IdeaPunch.ColorSlurp";
+  });
+}
+
+export async function openColorSlurpUrl(url: string) {
+  if (!(await isColorSlurpInstalled())) {
+    await showToast({
+      style: Toast.Style.Failure,
+      title: "ColorSlurp isn't installed",
+      message: "Install it from: https://colorslurp.com",
```

Comment:
one thing I realised just after merging - [Toasts](https://developers.raycast.com/api-reference/feedback/toast#toast) allow you to specify a `primaryAction` and `secondaryAction` that the user can invoke. So something like this

```suggestion
      message: "You can download it from the website or the app store",
      primaryAction: { title: "Open Homepage", onAction: () => open("https://colorslurp.com") },
      secondaryAction: {
        title: "Open App Store",
        onAction: () => open("https://apps.apple.com/us/app/colorslurp/id1287239339#?platform=mac"),
      },
```

would show up like this:

<img width="824" alt="image" src="https://user-images.githubusercontent.com/1155589/185736184-024f5a59-51a5-4383-9c7f-76bd97e3e2bd.png">

Not a big deal, but would make it ever so slightly simpler for users to access your website. 😸

Reply:
Oh that's super cool! Should I create a new pull request?

Reply:
go for it! 😉